src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To-Do List</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="to-do-list-con" id="todo-list">to do</div>
```

Comment:
- [ ] Nice work implementing the list of tasks :clap: However it is missing the input and the button to add a task. Kindly follow the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) and make sure your lists looks almost identical to [it](https://www.youtube.com/watch?v=AcUd-_Yjjqg) as well
![Screenshot from 2023-06-16 00-08-58](https://github.com/najibullahjafari/To-do-list/assets/91142432/3dcfede1-4bb5-4b67-a918-29affc6ceb79)